contrib/platform/src/com/sun/jna/platform/win32/ShellAPI.java

```diff
@@ -209,4 +211,852 @@ protected List getFieldOrder() {
+		 * Initiates a search starting from the specified directory.
+		 * </p>
+		 * </dd>
+		 * <dt>
+		 * <p>
+		 * <strong>open</strong>
+		 * </p>
+		 * </dt>
+		 * <dd>
+		 * <p>
+		 * Opens the file specified by the <strong>lpFile</strong> parameter.
+		 * The file can be an executable file, a document file, or a folder.
+		 * </p>
+		 * </dd>
+		 * <dt>
+		 * <p>
+		 * <strong>print</strong>
+		 * </p>
+		 * </dt>
+		 * <dd>
+		 * <p>
+		 * Prints the document file specified by <strong>lpFile</strong>. If
+		 * <strong>lpFile</strong> is not a document file, the function will
+		 * fail.
+		 * </p>
+		 * </dd>
+		 * <dt>
+		 * <p>
+		 * <strong>properties</strong>
+		 * </p>
+		 * </dt>
+		 * <dd>
+		 * <p>
+		 * Displays the file or folder's properties.
+		 * </p>
+		 * </dd>
+		 * </dl>
+		 */
+		public WString lpVerb;
+	
+		/**
+		 * <p>
+		 * Type: <strong>LPCTSTR</strong>
+		 * </p>
+		 * <p>
+		 * The address of a null-terminated string that specifies the name of
+		 * the file or object on which will perform the action specified by the
+		 * <strong>lpVerb</strong> parameter. The system registry verbs that are
+		 * supported by the <strong>ShellExecuteEx</strong> function include
+		 * "open" for executable files and document files and "print" for
+		 * document files for which a print handler has been registered. Other
+		 * applications might have added Shell verbs through the system
+		 * registry, such as "play" for .avi and .wav files. To specify a Shell
+		 * namespace object, pass the fully qualified parse name and set the
+		 * <strong>SEE_MASK_INVOKEIDLIST</strong> flag in the
+		 * <strong>fMask</strong> parameter.
+		 * </p>
+		 * <div class="note"><strong>Note</strong>&nbsp;&nbsp;If the
+		 * <strong>SEE_MASK_INVOKEIDLIST</strong> flag is set, you can use
+		 * either <strong>lpFile</strong> or <strong>lpIDList</strong> to
+		 * identify the item by its file system path or its PIDL respectively.
+		 * One of the two values-<strong>lpFile</strong> or
+		 * <strong>lpIDList</strong>-must be set.</div>
+		 * <div class="note"><strong>Note</strong>&nbsp;&nbsp;If the path is not
+		 * included with the name, the current directory is assumed.</div>
+		 */
+		public WString lpFile;
+	
+		/**
+		 * <p>
+		 * Type: <strong>LPCTSTR</strong>
+		 * </p>
+		 * <p>
+		 * Optional. The address of a null-terminated string that contains the
+		 * application parameters. The parameters must be separated by spaces.
+		 * If the <strong>lpFile</strong> member specifies a document file,
+		 * <strong>lpParameters</strong> should be <strong>NULL</strong>.
+		 * </p>
+		 */
+		public WString lpParameters;
+	
+		/**
+		 * <p>
+		 * Type: <strong>LPCTSTR</strong>
+		 * </p>
+		 * <p>
+		 * Optional. The address of a null-terminated string that specifies the
+		 * name of the working directory. If this member is
+		 * <strong>NULL</strong>, the current directory is used as the working
+		 * directory.
+		 * </p>
+		 */
+		public WString lpDirectory;
```

Comment:
When I wrote the test, I used the pre-existing `static Shell32.INSTANCE` object to call `ShellExecuteEx()`. `Shell32.INSTANCE` is initialized with `W32APIOptions.UNICODE_OPTIONS` - and `Shell32` extends `ShellAPI`. `ShellAPI` doesn't have an `INSTANCE` object (and from what I can tell on MSDN that makes sense because it's just a header file, not a DLL).